protobuf-c.spec

```diff
@@ -47,16 +73,27 @@ make check
 make install DESTDIR=$RPM_BUILD_ROOT
 rm -f $RPM_BUILD_ROOT/%{_libdir}/libprotobuf-c.la
 
+%if 0%{?suse_version} >= 01315
+%post -n lib%{name}1 -p /sbin/ldconfig
+%postun -n lib%{name}1 -p /sbin/ldconfig
+%else
 %post -p /sbin/ldconfig
 %postun -p /sbin/ldconfig
+%endif
 
+%if 0%{?suse_version} >= 1315
+%files -n lib%{name}1
+%else
 %files
+%endif
 %{_libdir}/libprotobuf-c.so.*
 %doc TODO LICENSE ChangeLog
 
+%if 0%{?suse_version} == 0
```

Comment:
Is this actually just an empty subpackage on all platforms?  Should it just be removed?

Reply:
It looks like it is empty on all platforms, as per my comment above.